crypto/fipsmodule/rand/entropy_pool_test.cc

```diff
@@ -0,0 +1,190 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include "internal.h"
+
+#include <gtest/gtest.h>
+
+#if defined(AWSLC_FIPS)
+
+// For standalone entropy pool tests, we add entropy through
+// RAND_entropy_pool_add(). The reason is that if a valid RAND_entropy_pool_get()
+// request cannot be satisfied, the "depleted" workflow will add entropy to the
+// thread-local entropy pool, not the locally initialized pool in the standalone
+// tests below.
+
+const size_t entropy_pool_size_halfed = ENTROPY_POOL_SIZE / 2;
+const uint8_t zero_entropy_pool[ENTROPY_POOL_SIZE] = {0};
+
+static void fill_fake_entropy(uint8_t fake_entropy_buffer[ENTROPY_POOL_SIZE],
+  char ch);
```

Comment:
```suggestion
```

Reply:
huh - must be a left-over from before I made it static or something...